src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,16 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500128
+name: cranial temporal crest
+def: "A dorsally concave, laterally-directed crest that terminates at the caudal-dorsal process of a zygomatic bone. It is bounded by the superior and inferior temporal lines." [DOI:10.5455/ijlr.20170908094210]
+synonym: "temporal crest" BROAD []
+intersection_of: UBERON:4200133 ! crest
+intersection_of: part_of UBERON:0001678 ! temporal bone
```

Comment:
From https://academic.oup.com/asj/article/29/2/158/270420?login=true:

<img width="524" alt="Screenshot 2023-02-10 at 10 29 40" src="https://user-images.githubusercontent.com/95643797/218077051-04b38f5f-d960-4d28-9350-e49d9df308b4.png">

and Netter's illustration from https://pocketdentistry.com/7-temporal-and-infratemporal-fossae/:
![ch07_u07-02-9781437726633-884638337](https://user-images.githubusercontent.com/95643797/218077463-3059680f-1ef4-4b93-a55b-f8dd87cb9dd3.jpeg)

At least in humans, all parts of the temporal crest are **not** part of the temporal bone. Your dbxref focuses on _Sus scrofa_, so if this assertion is true in that species, a taxon restraint would be necessary.

Reply:
What is EA?

So the issue here is the EQ? Maybe it be better to model it after temporal fossa, so that the subclass (no EQ) would be:
'crest'
'part of' some 'cranium' (note that cranium excludes mandible, unlike skull)